apiserver/machinemanager/state.go

```diff
@@ -50,3 +57,36 @@ func (s stateShim) AddMachineInsideNewMachine(template, parentTemplate state.Mac
 func (s stateShim) AddMachineInsideMachine(template state.MachineTemplate, parentId string, containerType instance.ContainerType) (*state.Machine, error) {
 	return s.State.AddMachineInsideMachine(template, parentId, containerType)
 }
+
+func (s stateShim) GetModel(tag names.ModelTag) (Model, error) {
+	m, err := s.State.GetModel(tag)
+	if err != nil {
+		return nil, err
+	}
+	return m, nil
+}
+
+func (s stateShim) Cloud(cloudName string) (cloud.Cloud, error) {
```

Comment:
these Cloud* methods aren't needed? stateShim embeds State, and State already has those methods